### PR TITLE
Separate rest api version from url definition in endpoint config

### PR DESCRIPTION
**Pull request by PiotrZakrzewski**

### CODE REVIEW

File: src/app/endpoints/selectedCohort/cohort-selection.service.js [hunk 1]
```diff
@@ -6,15 +6,25 @@
  * @name CohortSelectionService
  * @description handles cohort chart creation and user-interaction
  */
-angular.module('transmartBaseUi')
+angular.module('tmEndpoints')
     .factory('CohortSelectionService', [function () {
```
Comment (beaugogh) [inline src/app/endpoints/selectedCohort/cohort-selection.service.js]: is this the final fix or you plan to push more changes?

Comment (PiotrZakrzewski) [reply src/app/endpoints/selectedCohort/cohort-selection.service.js]: it is my final one. So feel free to comment ;)

Comment (beaugogh) [reply src/app/endpoints/selectedCohort/cohort-selection.service.js]: as long as it is another module, everything is fine :)

**Comment by beaugogh** [comment]
don't forget to rebase

File: src/app/endpoints/selectedCohort/CohortSharingService.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Created by piotrzakrzewski on 26/09/2016.
+ */
+'use strict'
+
+angular.module('tmEndpoints')
```
Comment (kretep) [inline src/app/endpoints/selectedCohort/CohortSharingService.js:11]: So did this file lose the ngdoc?

**Pull request merged**